.storybook/babel.config.js

```diff
@@ -7,17 +7,19 @@
 
 const path = require('path');
 const { readdirSync } = require('fs');
-const storybookBabelConfig = require('@storybook/core/dist/server/common/babel');
+const storybookBabelConfig = require('@storybook/core-common/dist/cjs/utils/babel');
```

Comment:
`@storybook/core` was split into `core-server`, `core-client`, and `core-common` in https://github.com/storybookjs/storybook/pull/13808

This updates the require statement to point to the new location of this file in Storybook version `6.2+`.